phpBB/posting.php

```diff
@@ -29,26 +26,26 @@
 
 
 // Grab only parameters needed here
-$post_id	= $request->variable('p', 0);
-$topic_id	= $request->variable('t', 0);
-$forum_id	= $request->variable('f', 0);
-$draft_id	= $request->variable('d', 0);
-$lastclick	= $request->variable('lastclick', 0);
-
-$preview	= (isset($_POST['preview'])) ? true : false;
-$save		= (isset($_POST['save'])) ? true : false;
-$load		= (isset($_POST['load'])) ? true : false;
-$confirm	= $request->is_set_post('confirm');
-$cancel		= (isset($_POST['cancel']) && !isset($_POST['save'])) ? true : false;
-
-$refresh	= (isset($_POST['add_file']) || isset($_POST['delete_file']) || isset($_POST['cancel_unglobalise']) || $save || $load || $preview);
+$post_id    = $request->variable('p', 0);
+$topic_id   = $request->variable('t', 0);
+$forum_id   = $request->variable('f', 0);
+$draft_id   = $request->variable('d', 0);
+$lastclick  = $request->variable('lastclick', 0);
+
+$preview    = (isset($_POST['preview'])) ? true : false;
+$save       = (isset($_POST['save'])) ? true : false;
+$load       = (isset($_POST['load'])) ? true : false;
+$confirm    = $request->is_set_post('confirm');
+$cancel     = (isset($_POST['cancel']) && !isset($_POST['save'])) ? true : false;
+
+$refresh    = (isset($_POST['add_file']) || isset($_POST['delete_file']) || isset($_POST['cancel_unglobalise']) || $save || $load || $preview);
 $submit = $request->is_set_post('post') && !$refresh && !$preview;
-$mode		= $request->variable('mode', '');
+$mode       = $request->variable('mode', '');
```

Comment:
please avoid this kind of changes. It adds confusion and lead to complex refactoring and conflicts with other PRs